pom.xml

```diff
@@ -74,6 +74,11 @@
             <version>3.8.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
Yeah i've been trying to find a RateLimiter library so i can replay the guava one, but haven't got any luck. Do you have anything in your mind that i could use?